webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -22,6 +24,16 @@ class ManageManifestModal extends Component {
     this.state = {
       redhat_repository_url: null,
     };
+
+    /**
+     * the Fills need to exist before adding the Slot,
+     * calling it on DidMount was too late as the Slot was already created.
+     * TODO: move these 2 components into a different file and connect them to redux,
+     * later register the fills in a global_js file or when the app launches.
+      */
+    const { ProviderDetails, SubscriptionManifest } = this;
+    addGlobalFill('katello-manage-manifest-form', 'ProviderDetails', <ProviderDetails key={1} />, 200);
+    addGlobalFill('katello-manage-manifest-form', 'SubscriptionManifest', <SubscriptionManifest key={2} />, 100);
```

Comment:
@amirfefer does it makes sense to register those on the constructor before the Slot is created?
otherwise the test didn't pass as there is an error when the Slot exist before the Fills.

Reply:
IMHO it doesn't seem like a best practice especially due to the enormous file size, and it requires a future refactoring.
How about just adding a slot for now to extend the manifest modal? especially due to the upcoming code freeze I think it saves us for refactoring this page atm.